shuup/front/checkout/confirm.py

```diff
@@ -97,6 +100,29 @@ def get_context_data(self, **kwargs):
         context["product_ids"] = ','.join(self._get_product_ids())
         return context
 
+    def _get_guest_order_data_with_gdpr(self):
+        basket = self.basket
+        first_name = basket._billing_address.name
```

Comment:
You really can't call `basket.billing_address` here https://github.com/shuup/shuup/blob/master/shuup/core/basket/objects.py#L271? Usually these private attributes should not be called outside the class.